File: 2021/06/21/main.go
package main

func generate(numRows int) [][]int {
	r := [][]int{}
	for i := 1; i <= numRows; i++ {
		l := make([]int, i)
		for j := range l {
			l[j] = 1
		}
		for j := 1; j+1 < i; j++ {
			l[j] = r[i-2][j-1] + r[i-2][j]
		}
		r = append(r, l)
	}
	return r
}

File: 2021/06/21/main.py
class Solution:
    def generate(self, numRows: int) -> List[List[int]]:
        r = []
        for i in range(numRows):
            l = [1 for _ in range(i+1)]
            for j in range(1, len(l) - 1):
                l[j] = r[i-1][j-1] + r[i-1][j]
            r.append(l)
        return r